CHANGELOG.md

```diff
@@ -28,6 +28,95 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+<!-- dumped content start -->
+
+## Unreleased
+
+### [shopsys/coding-standards]
+
+#### Added
+
+- [#384] [coding-standards] added new rule to keep class spacing consistent, Thanks to [@TomasVotruba]
+### [shopsys/framework]
```

Comment:
there should be an empty line above the heading - otherwise the output is malformated